packages/owl-bot/src/scan-googleapis-gen-and-create-pull-requests.ts

```diff
@@ -71,7 +74,20 @@ export async function scanGoogleapisGenAndCreatePullRequests(
     const stackSize = todoStack.length;
     for (const repo of repos) {
       octokit = octokit ?? (await octokitFactory.getShortLivedOctokit());
-      if (!(await copyExists(octokit, repo, commitHash, logger))) {
+      // Compare the config's begin-after-commit-hash and this commit hash,
+      // and don't create a pull request if this commit hash is older.
+      const repoString = repo.toString();
```

Comment:
I'd be tempted to encapsulate this bit in a private helper method.

Reply:
Done.